docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -906,14 +946,12 @@ data as more accurate than the street name part.
 
 To add noise to the DOB data, I will approximately follow the
 frequencies that Buzz Campbell measured in his BHDS unduplication
-work: Exact match for 96% of DOB, 2 of 3 fields exactly match for 3%,
-day and month fields transposed for 0.2%.  He found 0.24% missing, but
-I will leaving missing for outside of the DOB noise method for now,
-and perhaps make it a configurable option in the future.  Actually, it
-could be sensible to make all of these values configurable options in
-the future.
-
-2.3.10 Twins and multiparous births (16)
+work: Exact match for 96.11% of DOB, 2 of 3 fields exactly match for
+3.20%, no match for 0.26%, missing for 0.24%, day and month fields
+transposed for 0.18%. For future flexibility, I make all of these
+values configurable options.
```

Comment:
this appears to sum to 99.99%. was that on purpose/do we want to handle that last .001%? it could just be distributed between the others.

Reply:
It must be a rounding error; I think the way I implemented it, the extra goes to the exact match category.